WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.kt

```diff
@@ -2540,6 +2542,36 @@ class EditPostActivity : BaseAppCompatActivity(), EditorFragmentActivity, Editor
                                 openMediaLibrary(mediaType, initialSelection)
                             }
                         })
+                        editorFragment?.onLogJsException(object : GutenbergView.LogJsExceptionListener {
+                            override fun onLogJsException(exception: GutenbergJsException) {
+                                val stackTraceElements = exception.stackTrace.map { stackTrace ->
+                                    JsExceptionStackTraceElement(
+                                        stackTrace.fileName,
+                                        stackTrace.lineNumber,
+                                        stackTrace.colNumber,
+                                        stackTrace.function
+                                    )
+                                }
+
+                                val jsException = JsException(
+                                    exception.type,
+                                    exception.message,
+                                    stackTraceElements,
+                                    exception.context,
+                                    exception.tags,
+                                    exception.isHandled,
+                                    exception.handledBy
+                                )
+
+                                val callback = object : JsExceptionCallback {
+                                    override fun onReportSent(success: Boolean) {
+                                        // Do nothing
+                                    }
+                                }
```

Comment:
I dislike placing all of this GBK-specific logic in this file. I imagine it would require a larger refactor to avoid this, and I feel less confident envisioning what that might resemble. As one may notice, this addition mimics similar logic around it. 

I welcome any thoughts of simple improvements, as well as long-term refactor ideas for decoupling the various editors. Transparently, the current Android editor logic is very confusing to me.